packages/binding-coap/src/coap.ts

```diff
@@ -29,16 +27,30 @@ export * from "./coap-client";
 export * from "./coaps-client-factory";
 export * from "./coaps-client";
 
-export class CoapOption {
-    public "cov:optionName": OptionName;
-    public "cov:optionValue": OptionValue;
+export type CoapMethodName = "GET" | "POST" | "PUT" | "DELETE" | "FETCH" | "PATCH" | "iPATCH";
+
+export type BlockSize = 16 | 32 | 64 | 128 | 256 | 512 | 1024;
+
+export type BlockSizeOptionValue = 0 | 1 | 2 | 3 | 4 | 5 | 6;
+
+export interface BlockWiseTransferParameters {
+    "cov:block2SZX"?: BlockSize;
+    "cov:block1SZX"?: BlockSize;
 }
 
-export type CoapMethodName = "GET" | "POST" | "PUT" | "DELETE" | "FETCH" | "PATCH" | "iPATCH";
+export interface QBlockWiseTransferParameters {
```

Comment:
`QBlockWiseTransferParameters` and `BlockWiseTransferParameters` have been merged in the latest draft, so there's just `BlockWiseTransferParameters` for both now.